ocs_ci/utility/utils.py

```diff
@@ -5917,3 +5918,44 @@ def get_noobaa_cli_config():
         return constants.ODF_CLI_LOCAL_PATH, "noobaa"
     else:
         return constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH, ""
+
+
+def apply_oadp_workaround(namespace):
+    """
+    Apply the hostpath WA for IBMCloud's
+    https://access.redhat.com/articles/5456281#known-issues-with-cloud-providers-and-hyperscalers-18
+
+    Args:
+        namespace (str): Namespace where oadp is running
+    """
+
+    from ocs_ci.ocs.resources.csv import CSV, get_csvs_start_with_prefix
+    from ocs_ci.utility import templating
+
+    oadp_hostpath_map = {
+        "FS_PV_HOSTPATH": "/var/lib/kubelet/pods",
+        "PLUGINS_HOSTPATH": "/var/lib/kubelet/plugins",
+    }
+    csv_list = get_csvs_start_with_prefix("oadp-operator", namespace=namespace)
+    if csv_list:
+        for csv in csv_list:
+            if "oadp-operator" in csv["metadata"]["name"]:
```

Comment:
This check is not needed because the list is fetched with matching name.

Reply:
okay let me fix it